core/include/gnuradio-4.0/Block.hpp

```diff
@@ -1356,6 +1431,72 @@ struct Block : protected std::tuple<Arguments...> {
             return workInternal(requested_work);
         }
     }
+
+    void
+    processMessages(MsgPortInNamed<"__Builtin"> &port, std::span<const Message> messages) {
+        fmt::print("{} got a message\n", self().unique_name);
```

Comment:
I guess this is a forgotten debug message. ;-)

Reply:
I added a TODO 'clean up debugging output' :)